src/ServiceControl.Persistence.RavenDB/DatabaseSetup.cs

```diff
@@ -16,35 +18,66 @@ public async Task Execute(CancellationToken cancellationToken)
             await CreateDatabase(settings.DatabaseName, cancellationToken);
             await CreateDatabase(settings.ThroughputDatabaseName, cancellationToken);
 
-            await IndexCreation.CreateIndexesAsync(typeof(DatabaseSetup).Assembly, documentStore, null, null, cancellationToken);
+            await UpdateDatabaseSettings(settings.DatabaseName, cancellationToken);
+            await UpdateDatabaseSettings(settings.ThroughputDatabaseName, cancellationToken);
 
-            var expirationConfig = new ExpirationConfiguration
-            {
-                Disabled = false,
-                DeleteFrequencyInSec = settings.ExpirationProcessTimerInSeconds
-            };
+            await IndexCreation.CreateIndexesAsync(typeof(DatabaseSetup).Assembly, documentStore, null, null, cancellationToken);
 
-            await documentStore.Maintenance.SendAsync(new ConfigureExpirationOperation(expirationConfig), cancellationToken);
+            await ConfigureExpiration(settings, cancellationToken);
```

Comment:
Does this have immediate effect, and not require a database reset that UpdateDatabaseSettings() is doing as well? (Same for Audit above.)

Reply:
It was in this order before, so I don't know if it takes effect immediately or not. I would hope so, but I'll check.